dags/landsat_scenes_sync/landsat_scenes_fill_gaps.py

```diff
@@ -188,30 +176,31 @@ def fill_the_gap(landsat: str, scenes_limit: int) -> None:
                 key=latest_report,
             )
 
+            # This should just use Pandas. It's already a dependency.
             missing_scene_paths = [
                 scene_path
                 for scene_path in gzip.decompress(missing_scene_file_gzip).decode("utf-8").split("\n")
                 if scene_path
             ]
 
-            limit = scenes_limit if scenes_limit else len(missing_scene_paths)
-
-            logging.info(f"missing_scene_paths {missing_scene_paths[0:10]}")
+            logging.info(f"Number of scenes found {len(missing_scene_paths)}")
+            logging.info(f"Example scenes: {missing_scene_paths[0:10]}")
 
-            logging.info(f"Number of scenes found {len(missing_scene_paths)} limited in - {limit}")
+            if scenes_limit is not None:
```

Comment:
Will this be `0` @carioca-au?

Seems to me it'll either be an `int` or `None`.